pages/community/[contractAddress]/index.tsx

```diff
@@ -0,0 +1,43 @@
+import GalleryRoute from 'scenes/_Router/GalleryRoute';
+import CommunityPageScene from 'scenes/CommunityPage/CommunityPage';
+import { GetServerSideProps } from 'next';
+import GalleryRedirect from 'scenes/_Router/GalleryRedirect';
+import { MetaTagProps } from 'pages/_app';
+
+type CommunityPageProps = MetaTagProps & {
+  contractAddress?: string;
+};
+
+export const ENABLED_CONTRACTS = [
+  '0x5180db8f5c931aae63c74266b211f580155ecac8', // Crypto Coven
+  '0xb228d7b6e099618ca71bd5522b3a8c3788a8f172', // Poolsuite Exec
+  '0x123214ef2bb526d1b3fb84a6d448985f537d9763', // Poolsuite Pool
+  '0xa7d8d9ef8d8ce8992df33d8b8cf4aebabd5bd270', // Crypto Citizens
```

Comment:
i believe we were gonna do zeneca as well? or nah

Reply:
also two separate contracts for pool vs exec is interesting- good evidence that we'll def want some hierarchy in the future (zeneca also has 2 different contracts i believe)

Reply:
we can do zeneca why not. initially just had the 3 (coven, pool and citizens) on list

Reply:
looks like Zeneca is 1 contract with 2 erc 1155 tokens 1